CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## [Unreleased]
 
+### Added
+- New API action: SendCtrlAltDel, used to initiate a graceful shutdown,
```

Comment:
I would rename the customer facing name to `ShutdownVM` or something similar, because that's the customer visible behaviour. We should also explain/mention in a doc somewhere how this works, why it does, and when it doesn't (this is mostly  related to the kernel cmdline parameters being necessary I guess). This comment also applies to other places where `CtrlAltDelSomething` is going to end up being visible to the user.

Reply:
I considered that, but the fact is that reboot on CTRL+ALT+DEL is just a convention. It's completely up to the guest to decide what to do. A glaring example of unaligned behavior is the Windows "press CTRL+ALT+DEL to login" uhm... peculiarness.

To me, `ShutDownMicroVM` implies an actual, power-related, shut down action. Like ACPI. I.e. an action whose treatment by the guest is not debatable.

Not to say that I'm not a bit conflicted by this, however. I think we should have a vote on this one.